backend/src/main/java/ai/verta/modeldb/monitoring/RequestLatencyResource.java

```diff
@@ -9,6 +9,9 @@
           .labelNames("grpc_method")
           .name("verta_backend_requests_latency_seconds")
           .help("Request latency in seconds.")
+          .buckets(
+              0.2, 0.4, 0.6, 0.8, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4,
+              3.6, 3.8, 4.0)
```

Comment:
What about this???

Reply:
The default histogram for prometheus only uses 10 buckets with a wide spread.  This bucket distribution makes it much easier to see fine-grained changes in the performance data.